src/Training.TruckWorld.Clone.Backend/TruckWorld.Infrastructure/Common/Mapper/NotificationMessageMapper.cs

```diff
@@ -0,0 +1,10 @@
+﻿using AutoMapper;
+
+namespace TruckWorld.Infrastructure.Common.Mapper;
+
+public class NotificationMessageMapper : Profile
```

Comment:
Is it supposed to map from notification message to sms message and email message ?